src/docs/README.adoc

```diff
@@ -692,6 +692,11 @@ git bisect start good bad
 git bisect run ./bisect.sh
 ----
 
+Note::
```

Comment:
did you make a GH issue for this to be fixed as well? If not please do so. And link to the GH issue from the note in the readme.